roles/cloudman-boot/tasks/helm.yaml

```diff
@@ -1,18 +1,8 @@
-- name: Setup service account
-  shell: /usr/local/bin/kubectl create serviceaccount --namespace kube-system tiller && /usr/local/bin/kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-
-- name: Init Helm
-  shell: /usr/local/bin/helm init --service-account tiller
-
-- name: Wait for tiller pod to start...
-  command: /usr/local/bin/helm list
-  register: tiller_ready
-  until: tiller_ready.rc == 0
-  retries: 30
-  delay: 2
-
 - name: Add CloudVE Helm repo
   shell: /usr/local/bin/helm repo add galaxyproject https://raw.githubusercontent.com/cloudve/helm-charts/master/
 
+- name: Add CloudVE Helm repo
```

Comment:
Task name is incorrect and it would be better to add this first

Reply:
Changed to stable and moved up.